### ISSUE TITLE
C++ Standard

### ISSUE THREAD

**Issue by tbeu** [vonox7/golfcoder#21]
It seems that C++17, C++20, C++23 feature are not supported. Well, just saying.

**Comment by vonox7** [comment]
Yes, you are right. It seems like OneCompiler supports only C++ 201402. I'll have a look to other online compilers that have an API which support newer C++. @tbeu do you have any recommendation which one to use? I found now mycompiler.io, which seems to use at least C++ 201703. (Tested via the `__cplusplus` macro)

**Comment by vonox7** [comment]
OneCompiler updated their C++ library to C++17 (`__cplusplus` macro is "201703"). Sadly still no "up to date" C++ compiler.
